src/graph/GoExecutor.cpp

```diff
@@ -1136,6 +996,7 @@ bool GoExecutor::processFinalResult(RpcResponse &rpcResp, Callback cb) const {
                         }
                         return isReversely() ? srcId : dstId;
```

Comment:
This definition actually made users confused, as they talk about this in issue#1600. And it makes conflict with the definition of `GO BIDIRECT`. And i have change it in pr#1694. I will change the implementation of #1694 after this pr being merged.

Reply:
That's good.